pkgs/sops-install-secrets/main.go

```diff
@@ -936,11 +957,15 @@ func installSecrets(args []string) error {
 				return fmt.Errorf("Cannot write key to age file: %w", err)
 			}
 		}
+		ageFile.Close()
 	}
 
 	if err := decryptSecrets(manifest.Secrets); err != nil {
 		return err
 	}
+	if len(manifest.SSHKeyPaths) != 0 {
+		keyring.Remove()
```

Comment:
Maybe even the whole decryptSecrets part needs to be it's own function to reduce the scope of `keyring`.

Reply:
After checking `decryptSecrets`, we can just move setting up the keyring and importing the agefile into this function.